internal/databases/mysql/mysql.go

```diff
@@ -37,6 +37,23 @@ type Backup struct {
 	*internal.Backup
 }
 
+func scanToMap(rows *sql.Rows, dst map[string]interface{}) error {
```

Comment:
Please, use StructScan http://jmoiron.github.io/sqlx/#advancedScanning

Reply:
Wal-g doesn't depend on sqlx now.  It doesn't make sense to bring a new dependency for just one call.